tools/api-markdown-documenter/package.json

```diff
@@ -89,31 +89,30 @@
 	},
 	"devDependencies": {
 		"@biomejs/biome": "^1.7.3",
-		"@fluid-internal/mocha-test-setup": "~2.23.0",
+		"@fluid-internal/mocha-test-setup": "~2.60.0",
 		"@fluid-tools/markdown-magic": "file:../markdown-magic",
 		"@fluidframework/build-common": "^2.0.3",
-		"@fluidframework/build-tools": "^0.57.0",
+		"@fluidframework/build-tools": "^0.58.3",
 		"@fluidframework/eslint-config-fluid": "^6.0.0",
 		"@microsoft/api-extractor": "^7.52.11",
 		"@types/chai": "^5.2.1",
 		"@types/hast": "^3.0.4",
 		"@types/mdast": "^4.0.4",
-		"@types/mocha": "^10.0.1",
+		"@types/mocha": "^10.0.10",
 		"@types/node": "^18.19.0",
 		"c8": "^10.1.3",
-		"chai": "^5.2.1",
-		"concurrently": "^8.2.1",
+		"chai": "^6.0.1",
+		"concurrently": "^9.2.1",
 		"copyfiles": "^2.4.1",
-		"cross-env": "^7.0.3",
-		"dir-compare": "^4.0.0",
-		"eslint": "~8.55.0",
-		"eslint-plugin-chai-expect": "^3.0.0",
-		"eslint-plugin-chai-friendly": "^0.7.2",
-		"mocha": "^10.8.2",
+		"cross-env": "^10.0.0",
+		"dir-compare": "^5.0.0",
+		"eslint": "~8.57.0",
+		"eslint-plugin-chai-expect": "^3.1.0",
+		"eslint-plugin-chai-friendly": "^1.1.0",
+		"mocha": "^11.7.2",
 		"mocha-multi-reporters": "^1.5.1",
-		"prettier": "~3.0.3",
```

Comment:
Prettier went away?

Reply:
Forgot to call that out in the PR description (updated). This package uses `biome` for formatting. The `prettier` dep was unused.